contracts/AccessControl/RoleBasedAccessControl.sol

```diff
@@ -97,7 +97,7 @@ contract RoleBasedAccessControl is IAccessPolicy, AccessControlled {
             bool grantedLocal = localAccess == TriState.True;
 
             // Log and return
-            Access(subject, role, object, verb, grantedLocal);
+            Access(subject, role, object, verb, granted);
```

Comment:
Where does `granted` come from?

Reply:
In PR #39 this is refactored, this will result in a merge conflict. When this PR is merged, I can rebase #39 on top of it.

Reply:
my fault when putting parameters back into order! there is no error for unknown variable or no warning for using uninitialized one, so it is a little weird it compiles.

Reply:
Solidity allocates all local variables of a scope at the start of the scope, `bool granted` is declared later in this scope. That's why it compiles. I think it shouldn't though. The value will be undefined.